pages/sdk/api-kit.mdx

```diff
@@ -136,8 +147,14 @@ For more detailed information, see the guide [Integrating the Protocol Kit and A
   In this step we need to sign the transaction with the Protocol Kit and submit the signature to the Safe Transaction Service using the `confirmTransaction` method.
 
   ```typescript
+  const protocolKitOwner2 = await Safe.create({
+    provider: RPC_URL,
+    signer: OWNER_1_PRIVATE_KEY,
```

Comment:
This is owner 2 private key. You can use the connect method as well instead creating a new instance

Reply:
Will fix it. Using the connect will probably create confusion here as this code will probably executed in a different user session